feat: use templates while creating features and issues (#611)
getAlby/lightning-browser-extension: SECURITY.md

# Security Policy

## Supported Versions

See releases section for versions of Alby that are currently being supported with 

> security updates: https://github.com/getAlby/lightning-browser-extension/tags

Reviewer: ```suggestion
security updates: https://github.com/getAlby/lightning-browser-extension/releases
```

Author: oops! thanks for pointing this out. Resolved!

## Reporting a Vulnerability

> If you found a security vulnerability on this platform please send an email to hello@michaelbumann.com (do not use this email for asking any kind of support).

Reviewer: ```suggestion
If you found a security vulnerability on this platform please send an email to security@getalby.com (do not use this email for asking any kind of support).
```

Author: shall I remove the PGP fingerprint then? it's for hello@michaelbumann.com

Author: resolved

The following key may be used to communicate sensitive information to Alby developers:

| Name | PGP Fingerprint |
|------|-------------|

> | Michael Bumann | 60E0 4E32 C48B 6900 41ED 99E4 8ABD EEC3 CEF8 5746 |

Reviewer: ```suggestion
| Michael Bumann | 163A 86B4 B913 FF60 A84E  2AB8 FFE0 5711 E1D0 A2B0 |
```

> | Michael Bumann | 60E0 4E32 C48B 6900 41ED 99E4 8ABD EEC3 CEF8 5746 |

Reviewer: ```suggestion
| Michael Bumann | 60E0 4E32 C48B 6900 41ED 99E4 8ABD EEC3 CEF8 5746 |
| security@getalby.com | B4A4 1AFC 9A15 0CCE A409  0D78 948A 7A9B 1B51 CDAE |
```

> | Michael Bumann | 163A 86B4 B913 FF60 A84E  2AB8 FFE0 5711 E1D0 A2B0 |

Reviewer: ```suggestion
| Michael Bumann | 163A 86B4 B913 FF60 A84E  2AB8 FFE0 5711 E1D0 A2B0 |
| security@getalby.com | B4A4 1AFC 9A15 0CCE A409  0D78 948A 7A9B 1B51 CDAE |
```

# Security Policy

## Supported Versions

See releases section for versions of Alby that are currently being supported with 
security updates: https://github.com/getAlby/lightning-browser-extension/releases

## Reporting a Vulnerability

If you found a security vulnerability on this platform please send an email to security@getalby.com (do not use this email for asking any kind of support).

The following key may be used to communicate sensitive information to Alby developers:

| Name | PGP Fingerprint |
|------|-------------|
| Michael Bumann | 163A 86B4 B913 FF60 A84E  2AB8 FFE0 5711 E1D0 A2B0 |
| security@getalby.com | B4A4 1AFC 9A15 0CCE A409  0D78 948A 7A9B 1B51 CDAE |

> We also accept vulnerability reports through https://huntr.dev platform.

Reviewer: Do we need to do here anything to set this up? I actually don't know how huntr.dev works.

Author: If you're not registered with us, we will try our best to reach you privately. If we are unable, we request that you create a SECURTIY.md on your repository, with a valid e-mail address. After this, we can contact you when we receive new disclosures against your repository.

Following this, you will have the option to validate or invalidate disclosures. If you choose to validate the disclosure, the reporter will immediately receive a bounty reward.

Afterwards, we encourage you to submit a fix for the vulnerability, where we request the patch commit SHA.

If you are interested in receiving more security disclosures, you can let researchers know they can win bounties protecting your repository by copying this small code snippet into your README.md:

[![huntr](https://cdn.huntr.dev/huntr_security_badge_mono.svg)](https://huntr.dev)

Author: I don't think anything is to be done. however, a Github account can signup as a maintainer to receive instant notification.
 if the disclosure form is submitted by anyone against our repo. this will give bounty reward to the submitter and fix reward if maintainer registered on website fixes himself that bounty

Reviewer: ok, nice! thanks! sounds great!
